components/logins/src/error.rs

```diff
@@ -152,7 +152,7 @@ impl GetErrorHandling for LoginsError {
                 ErrorHandling::convert(LoginsStorageError::UnexpectedLoginsStorageError(
                     "must be an empty DB to migrate".to_string(),
                 ))
-                .report_error("logins:migration")
```

Comment:
This doesn't prevent me from approving, but why were these changes from `:` to `-` were made?

Reply:
Sentry ergonomics: If you type ":" into the Sentry search box, it interprets that as filtering based on a field which means you need to quote the strings in order to search for them.

Reply:
Makes sense--thanks!